Codegen/test/TestTypeCobolDomainCodegen.cs

```diff
@@ -0,0 +1,1034 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.IO;
+using TypeCobol.Codegen.Config;
+using TypeCobol.Codegen.Skeletons;
+using TypeCobol.Compiler; // DocumentFormat
+using TypeCobol.Compiler.Domain;
+using TypeCobol.Compiler.Parser;
+using TypeCobol.Tools; // CodeElementDiagnostics
+using TypeCobol.Tools.Options_Config;
+
+namespace TypeCobol.Codegen.Domain
+{
+    [TestClass]
+    public class TestTypeCobolDomainCodegen
```

Comment:
Can you combine this class with TestTypeCobolCodegen and just surround TestInitialize and TestCleanup with `#if DOMAIN_CHECKER` like you did with GrammarTest ?

Reply:
No beacuse these are different tests, this will means that I only test Semantic Domain if and only if DOMAIN_CHECKER is defined, in the future using DOMAIN_CHECKER should be removed.

Reply:
Sorry My mistake, yes I think we can do that, I will do that afterwards.